src/plugins/fs/generic.c

```diff
@@ -82,6 +82,7 @@ static const BDFSInfo fs_info[] = {
     {"f2fs", "fsck.f2fs", "fsck.f2fs", "resize.f2fs", BD_FS_OFFLINE_GROW | BD_FS_OFFLINE_SHRINK, NULL, "dump.f2fs", NULL},
     {"reiserfs", "reiserfsck", "reiserfsck", "resize_reiserfs", BD_FS_ONLINE_GROW | BD_FS_OFFLINE_GROW | BD_FS_OFFLINE_SHRINK, "reiserfstune", "debugreiserfs", "reiserfstune"},
     {"nilfs2", NULL, NULL, "nilfs-resize", BD_FS_ONLINE_GROW | BD_FS_ONLINE_GROW, "tune-nilfs", "tune-nilfs", "tune-nilfs"},
+    {"exfat", "fsck.exfat", "fsck.exfat", NULL, 0, "exfatlabel", "dumpexfat", NULL},
```

Comment:
Doesn't match with `tune.exfat` called above. Is that on purpose or just leftover from the older tools?

Reply:
No, I forgot to rewrite this to the new tools, fixed.